examples/json-web-token.roc

```diff
@@ -0,0 +1,105 @@
+app [Model, server] { pf: platform "../platform/main.roc" }
+
+import pf.Stdout
+import pf.Http exposing [Request, Response]
+import pf.Jwt
+
+Model : {}
+
+server = { init, respond }
+
+respond : Request, Model -> Task Response [ServerErr Str]_
+respond = \_, _ ->
+
+    # We hardcode the JWT here and ignore the request for simplicity, but normally this is how
+    # you would decode the request body (assuming the token is in the body and not a URL param)
+    # ```
+    # exampleBody = Http.parseFormUrlEncoded req.body
+    # ```
+    exampleBody : Result (Dict Str Str) _
+    exampleBody =
+        Dict.empty {}
+        |> Dict.insert "id_token" "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.joReqPNNkWQ8zQCW3UQnhc_5NMrSZEOQYpk6sDS6Y-o"
```

Comment:
Not very important, but I believe there is `Dict.single`

Also, I would remove the `|> Ok`. That or commit and have full error handling with either `?` or `Task.fromResult!`

Reply:
Done in e8c55628c3042953f02061ead6d00229547d2cc1